cicd/bootstrap.sh

```diff
@@ -81,5 +81,19 @@ add_cicd_bin_to_path() {
 
 add_cicd_bin_to_path
 
-# log in to ephemeral cluster
-oc_wrapper login --token=$OC_LOGIN_TOKEN --server=$OC_LOGIN_SERVER
+# !!Hotswap based on availability
+login_to_available_server() {
+  status = $(curl -s -w 200 $OC_LOGIN_SERVER)
```

Comment:
this will always display `200`. 

try 
```
curl -s -w 200 foobar
```

the -w ( `--write-out` ) will always output what you added to STDOUT

need to be updated to something else, perhaps just evaluating an url without output, like:  

```
openshift_cluster_available() {

    local SERVER_URL=$1

    curl -s "$SERVER_URL" >/dev/null
}

if openshift_cluster_available "$OC_LOGIN_SERVER"; then
# [ ... ] 
```